views/place.hbs

```diff
@@ -1,54 +1,65 @@
+{{>header}}
+{{>options}}
 
-<ul class="">
-  <div class="">
-    <a href="/">{{localText.homePage}}</a>
-    <a href="/{{lang}}/places">{{localText.backList}}</a>
+<a class="f6 ma4 w4 tc grow no-underline br-pill ba bw1 pv2 dib mid-gray b gray" href="/{{lang}}/add-place">{{localText.add}}</a>
+<div class="pa4">
+  <div class="overflow-auto">
+    <table class="f6 w-100 mw8 center bb b--turquoise bw2" cellspacing="0">
+      <thead>
+        <tr class="ma2">
+          <th class="fw6 tl pa3">
```

Comment:
I think you should give the image `th` a fixed width maybe like the class `w25` so then it will look the same for all the different things -   atm I think the table changes how it looks depending on the size of the image or one how much text is in the fields of the database

Reply:
same for the events part as well